lib/LaTeXML/Core/Mouth.pm

```diff
@@ -281,6 +281,8 @@ sub readToken {
   while (1) {    # Iterate till we find a token, or run out. (use return)
                  # ===== Get next line, if we need to.
     if ($$self{colno} >= $$self{nchars}) {
+      my $lastn   = $$self{nchars};
+      my $lastcol = $$self{colno};
```

Comment:
`$lastn` and `$lastcol` are never used? Is there a missing commit?

Reply:
nope, missing cleanup. Ugh.